independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/BeanProcessor.java

```diff
@@ -403,8 +413,25 @@ public Builder addRemovalExclusion(Predicate<BeanInfo> exclusion) {
             return this;
         }
 
+        /**
+         * 
+         * @param removeFinalForProxyableMethods
+         * @return self
+         * @deprecated This method will be removed at some point post Quarkus 1.4
+         */
```

Comment:
Add `@see `setRemoveFinalFromProxyableMethods(boolean)`.

Reply:
Did you mean `@see #setTransformUnproxyableClasses()`?

Reply:
Yea, that's what I meant :-D